Makefile

```diff
@@ -98,30 +97,14 @@ endif
 .PHONY: ci
 ci: c db-migrate
 ifeq ("$(package)", "all")
-    ifneq ("$(suite)", "new")
-		./hack/bin/cabal-run-integration.sh all
-    endif
-    ifneq ("$(suite)", "old")
-		make c package=integration
-		./hack/bin/cabal-run-integration.sh integration
-    endif
-else
-  ifeq ("$(package)", "integration")
+	./hack/bin/cabal-run-integration.sh all
+	./hack/bin/cabal-run-integration.sh integration
+endif
+ifeq ("$(package)", "integration")
 	./hack/bin/cabal-run-integration.sh integration
-  else
-    ifeq ("$(suite)", "old")
-		./hack/bin/cabal-run-integration.sh $(package)
-    else
-      ifeq ("$(suite)", "new")
-		make c package=integration
-		./hack/bin/cabal-run-integration.sh integration
-      else
-		make c package=integration
-		./hack/bin/cabal-run-integration.sh $(package)
-		./hack/bin/cabal-run-integration.sh integration
-      endif
-    endif
-  endif
+else
+	make c package=$(package)
```

Comment:
This would call `make c` for the package a second time. The first time would be because of the `ci: c` line. I think the whole if can go away